packages/tasks/src/local-apps.ts

```diff
@@ -1,6 +1,12 @@
 import type { ModelData } from "./model-data";
 import type { PipelineType } from "./pipelines";
 
+interface Snippet {
```

Comment:
```suggestion
export interface LocalAppSnippet {
```

suggestion to export it and name it as `LocalAppSnippet` so that we can use it from hf.co codebase

Reply:
Fixed in https://github.com/huggingface/huggingface.js/pull/778/commits/7983478d5f136b2e101752b229fa906f06c719f1

Reply:
Actually, I like the visual improvements mentioned above and we could use them for other snippets as well (i.e., use in transformers), not just local apps. We can discuss about a `Snippet` type in a separate PR and move forward with this one for now.